frontend/src/contract-models.ts

```diff
@@ -42,7 +42,8 @@ export function characterFromContract(id: string | number, data: string[]): ICha
   const legs = data[7];
   const boots = data[8];
   const race = data[9];
-  return { id: +id, xp, level, trait, traitName, staminaTimestamp, head, arms, torso, legs, boots, race };
+  const version = 0;
```

Comment:
Version always 0 here?

Reply:
Just initliazed, it's fetched later. Couldn't get version returned via `characters.get(id)`, because of 'stak too deep' error. Also didn't want to create another map in storage so, just updating the version here instead.